mlir/test/Quantum/WrapperTest.mlir

```diff
@@ -0,0 +1,172 @@
+// Copyright 2022-2023 Xanadu Quantum Technologies Inc.
```

Comment:
New copyright notices should start with the current year
```suggestion
// Copyright 2023 Xanadu Quantum Technologies Inc.
```

Reply:
https://github.com/PennyLaneAI/catalyst/pull/106/commits/dc84c6003af6ebec6478642e7e363c594ee95e8f